bigtable/client/filters.h

```diff
@@ -196,14 +196,17 @@ class Filter {
   /**
    * Return a filter that only accepts the first @p n cells in a row.
    *
-   * Notice that cells might be repeated, such as when interleaving the results
-   * of multiple filters via the Union() function (aka Interleaved in the
-   * proto).  Furthermore, notice that this is the cells within a row, if there
-   * are multiple column families and columns, the cells are returned ordered
-   * by first column family, and then by column qualifier, and then by
-   * timestamp.
-   *
-   * TODO(#82) - check the documentation around ordering of columns.
+   * Note that cells might be repeated, such as when interleaving the results
+   * of multiple filters via the `Interleave()` function.  Furthermore, this
+   * filter apples to the cells within a row, if there are multiple column
+   * families and/or columns in a row the order is:
+   * - All the cells for a column family appear together, but there is no
+   *   guarantee on the order of the column families.  Furthermore, column
+   *   families may appear in different orders in different rows.
+   * - Within a column family the cells are ordered by column name, column names
```

Comment:
- Within a column family, the cells are ordered by column name, where column names are sorted lexicographically.

Reply:
Thanks!

Reply:
Fixed.